lib/matplotlib/category.py

```diff
@@ -116,14 +116,16 @@ def __init__(self, units_mapping):
         """
         Parameters
         -----------
-        units_mapping : Dict[str, int]
+        units_mapping : Dict[AnyStr, int]
```

Comment:
Usually we do not use type annotation syntax in the docstrings. I suppose this came in before we consistently adhered to numpydoc. I'd go for
```suggestion
        units_mapping : dict
            Mapping category names (str) to indices (int).
```